decidim-debates/spec/types/integration_schema_spec.rb

```diff
@@ -10,6 +10,19 @@
       fragment fooComponent on Debates {
         debate(id: #{debate.id}){
```

Comment:
and here

Reply:
done in [7c7ad42](https://github.com/decidim/decidim/pull/14358/commits/7c7ad42d2220e4d982785f81246a9e7418c82d76)